osu.Game/Skinning/SkinManager.cs

```diff
@@ -280,6 +287,8 @@ public Action<IEnumerable<Live<SkinInfo>>> PresentImport
 
         public Task<Live<SkinInfo>> Import(ImportTask task, bool batchImport = false, CancellationToken cancellationToken = default) => skinImporter.Import(task, batchImport, cancellationToken);
 
+        public void ExporCurrenttSkin() => CurrentSkinInfo.Value.PerformRead(s => skinExporter.ExportAsync(s));
```

Comment:
Typo in method name.

Also method should arguably not export "current skin", it should export a skin as given by a parameter.